File: TaskManagerApi/TaskManagerApi/Models/Task.cs
using System;

namespace TaskManagerApi.Models
{
    public class Task
    {
        public int TaskId { get; set; }
        public Nullable<int> ParentId { get; set; }
        public string Name { get; set; }
        public string StartDate { get; set; }
        public string EndDate { get; set; }
        public int Priority { get; set; }
        public string ParentTaskName { get; set; }
    }
}

File: TaskManagerAngular/src/app/task.ts
export class Task {
    TaskId: number = 0;
    ParentId!: number;
    Name: string = "";
    StartDate: string = new Date().toISOString().split('T')[0];
    EndDate: string = new Date().toISOString().split('T')[0];
    Priority: number = 0;
    ParentTaskName: string = "";
  }
export class  DropdownList
{
    value: number;
    label: string
}